docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>I'd refer again to the docs when mentioning <code>... using OTT's Meta OT</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>